### PR TITLE
Adds an article about XSS attacks and its prevention.

### PR DESCRIPTION
**Pull request by ilhan-mstf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ Supported by: [GuardRails.io](https://www.guardrails.io)
 - [Secure Coding Guidelines](https://www.oracle.com/technetwork/java/seccodeguide-139067.html) - Secure Coding Guidelines for Java SE
 - [Securing a Web Application](https://spring.io/guides/gs/securing-web/) - This guide walks you through the process of creating a simple web application with resources that are protected by Spring Security.
 - [Spring Security Guides](https://docs.spring.io/spring-security/site/docs/current/guides/html5/index.html) - Step by step guides on how to use Spring Security.
+- [Prevent cross-site scripting (XSS) attacks](https://www.ibm.com/developerworks/library/se-prevent-cross-site-scripting-attacks/index.html) - This article explains how XSS attacks work and suggests a methodoloy to block XSS attacks.
```
Comment (streichsbaer) [inline README.md]: There’s a typo in methodology.

**Review by streichsbaer — CHANGES_REQUESTED** [review]
Thanks for the contribution, @ilhan-mstf 

Please fix the typo, then it’s good to be merged!

File: README.md (hunk 1 above)
Comment (ilhan-mstf) [reply README.md]: Thanks for the feedback, I've just fixed the typo.

**Review by ghost — APPROVED** [review]
LGTM :shipit: Thanks for your contribution!

**Review by r0flc0pt4 — APPROVED** [review]
LGTM!

**Review by streichsbaer — APPROVED** [review]

**Pull request merged**